public/docs/ts/latest/guide/i18n.jade

```diff
@@ -0,0 +1,306 @@
+    * No information/state is (typically) lost. The user is presumably changing the language because he could not understand the earlier language.
+    This means that he doesn't have unsaved information in the application.
+    * Tools support: the strings to translate are easily extracted from our templates.
+    The resulting translations are in a generic format that can be consumed both by Angular 2 and one of the many translation softwares available.
+    * Cannot support multiple languages in the same application view.
+    * We have to reload the app to change the language.
+    * Extra server side support is needed: Since we generate different precompiled files for each language, the server must perform cookie/user agent analysis
+    to decide which localized version of the application code should be returned to the user. This also causes a cache miss.
+    * The server is now responsible for determining the default localized version to serve. (e.g. cookies / geo-ip / Accept-Language header, etc.)
+  
+
+a(id="angular2-i18n")
+.l-main-section
+:marked
+  ## Angular 2 and i18n
+  
+  Whatever approach we decide to use, we must update our templates to define which strings will be translated.
+ 
+  
+a(id="i18n-directive")
+.l-main-section
+:marked
+  ### i18n directive
+  
+  The i18n directive is what tells Angular where it should inject the translations.
+  It is used by both the JiT and the AoT approaches.
+  
+  We simply add the attribute `i18n` to an element and the content string will be replaced when necessary.
+  
++makeExample('i18n/ts-snippets/i18n.config.snippets.ts', 'i18n-directive')(format=".")
+
+:marked
+  To help the translators, we can add more information about the meaning and context of this string.
+  Simply add a description to the i18n attribute:
+  
++makeExample('i18n/ts-snippets/i18n.config.snippets.ts', 'i18n-directive-desc')(format=".")
+
+:marked
+  We can add some meaning as well, separate the meaning from the description with a pipe `|`:
+  
++makeExample('i18n/ts-snippets/i18n.config.snippets.ts', 'i18n-directive-meaning')(format=".")
+
+:marked
+  Both the meaning and the description will be extracted by our messages extractor and added to the messages file.
+  It will help our translators to translate our application with a better understanding of what our text means. 
+
+  
+a(id="i18n-plural-pipe")
+.l-main-section
+:marked
+  ### i18n plural pipe
+  
+  When we need to pluralize some terms based on values, we can use the plural pipe.
+  
+  This pipe provides different translations based on a number and on a mapping (which is an object
+  that mimics the [ICU format](http://userguide.icu-project.org/formatparse/messages)):
+  
++makeExample('i18n/ts-snippets/i18n.config.snippets.ts', 'i18n-plural-pipe')(format=".")
+
+.alert.is-important
+  :marked
+    The plural pipe is not yet supported by the ng-xi18n messages extractor. 
```

Comment:
ahh good to know, I did most of my tests with xliff since it's the default, I'll update the note then